### ISSUE TITLE
bug/feat(R): Fix standalone Activation layer handling in orbital_keras_dag_impl — HardSigmoid, HardSwish, and others treated as pass-through instead of applying activation

### ISSUE THREAD

**Issue by davidrsch** [davidrsch/orbital-r-fork#35]
## Summary

`R/ann-helpers.R`'s `activation_expr()` already implements `hardsigmoid`, `hard_sigmoid`, and `hardswish`/`hard_swish` for brulee models. However, `orbital_keras_dag_impl()` in `R/model-keras.R` does NOT dispatch standalone `layer_activation("hard_sigmoid")` or `layer_hard_sigmoid()` / `layer_activation("hard_swish")` keras3 layers as separate DAG nodes.

Currently, the catch-all `activation` branch (line 328) forwards names unchanged — the activation expression is **never applied** for standalone Activation layers that appear *outside* a Dense layer's inline activation. This affects any model using:

```r
inp |> layer_dense(16) |> layer_batch_normalization() |> layer_activation("hard_sigmoid")
```

The `layer_activation("hard_sigmoid")` step would be treated as a no-op (pass-through), silently producing wrong predictions.

## Parity

Python has `HardSigmoidTranslator` and `HardSwishTranslator` registered in `TRANSLATORS` (davidrsch/orbital-1). R must handle these as first-class DAG nodes, not pass-throughs.

## Acceptance Criteria

- Fix the catch-all `activation` branch in `orbital_keras_dag_impl()` to apply `activation_expr()` for any activation string found in the layer config, rather than forwarding names unchanged
- Specifically handle `HardSigmoid`, `HardSwish`, `ELU`, `GELU`, `LeakyReLU` as standalone `Activation` layers
- Add DAG trigger regex entries for `hardsigmoid|hardswish` 
- Numerical accuracy tests added for BatchNorm → HardSigmoid and BatchNorm → HardSwish patterns
- Tests skip if keras3 not installed

## Notes

- The bug also affects any other non-linear standalone Activation layer placed *after* a non-Dense layer (BatchNorm → Activation("relu") would silently skip the relu)
- Fix the `activation` pass-through to always look up the activation name in the keras3 layer config and call `activation_expr()`
- For parity: davidrsch/orbital-1#30 (Gelu op), existing `HardSigmoidTranslator` + `HardSwishTranslator` in Python

Related: davidrsch/orbital#28
